flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/calls/FunctionGenerator.scala

```diff
@@ -753,6 +753,15 @@ object FunctionGenerator {
     Seq(FLOAT, INTEGER),
     BuiltInMethods.TRUNCATE_FLOAT)
 
+  addSqlFunctionMethod(
+    JSON_QUERY,
+    Seq(VARCHAR, CHAR, RAW, RAW, RAW),
```

Comment:
Just curious, `SqlStdOperatorTable.JSON_VALUE` has 7 parameters, why here only 5?

Reply:
> Just curious, `SqlStdOperatorTable.JSON_VALUE` has 7 parameters, why here only 5?

`SqlStdOperatorTable. JSON_QUERY ` have only 5.

Reply:
Sorry for the noisy,  not notice it's `JSON_QUERY`,  instead of `JSON_VALUE`.